packages/atlas/src/components/NotificationTile/NotificationTile.styles.ts

```diff
@@ -0,0 +1,103 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { cVar, sizes } from '@/styles'
+
+type Variant = 'default' | 'compact'
+
+type NotificationWrapperProps = {
+  read?: boolean
+  selected?: boolean
+  loading?: boolean
+  variant: Variant
+}
+
+const getNotificationWrapperStyles = ({ read, selected, loading, variant }: NotificationWrapperProps) => {
+  if (selected) {
+    return css`
+      background-color: ${cVar('colorBackgroundElevated')};
+    `
+  }
+  if (!read) {
+    if (variant === 'default') {
+      return css`
+        background-color: ${cVar('colorBackground')};
+
+        :hover {
+          background-color: ${cVar('colorBackgroundStrong')};
+        }
+      `
+    }
+    return css`
+      background-color: ${cVar('colorBackgroundMutedAlpha')};
+
+      :hover {
+        background-color: ${cVar('colorBackgroundStrong')};
+      }
+    `
+  }
+  if (loading) {
+    return
+  }
+  if (variant === 'default') {
+    return css`
+      background-color: ${cVar('colorBackgroundMuted')};
+
+      :hover {
+        background-color: ${cVar('colorBackground')};
+      }
+    `
+  }
+  return css`
+    background-color: transparent;
+
+    :hover {
+      background-color: ${cVar('colorBackground')};
+    }
+  `
+}
+
+const getReadNotificationVariant = ({ read, variant }: NotificationWrapperProps) =>
+  !read &&
+  css`
+    ::after {
+      content: '';
+      display: block;
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: ${variant === 'default' ? 4 : 2}px;
+      height: 100%;
+      background-color: ${cVar('colorBackgroundPrimary')};
+    }
+  `
+
+export const Wrapper = styled.div<NotificationWrapperProps>`
```

Comment:
Im getting: 
![image](https://user-images.githubusercontent.com/51168865/155290735-58c6e00c-f83d-4d66-81ac-2dc759f15887.png)
I think it comes from here